### PR TITLE
Added "Commander" to "Cli Builders" list.

### PR DESCRIPTION
**Pull request by meskyanichi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
 
 ## Cli Builders
  * [clik](https://github.com/trans/clik) - Easy command line interfaces
+ * [commander](https://github.com/meskyanichi/commander) - Command-line interface builder for the Crystal programming language
```
Comment (veelenga) [inline README.md]: Please omit `for the Crystal programming language` here.

**Comment by veelenga** [comment]
Thanks for contributing :+1:. Please update and do git squash/amend.

**Comment by meskyanichi** [comment]
Done!

**Comment by veelenga** [comment]
Thanks :+1:

**Comment by meskyanichi** [comment]
Thanks for the merge! :)

**Pull request merged**